### ISSUE TITLE
Sparkjava 2.5 doesn't print stack trace to the console for 500 Internal ERROR but Sparkjava 2.2 Does!

### ISSUE THREAD

**Issue by SeunMatt** [perwendel/spark#679]
Hello,
 Was working on a comprehensive tutorial of using sparkjava for web apps. Then I found out that 500 Internal Errors' stack trace are not printed neither are they logged on the server-side. They don't show up on the console.

This code will print stack trace and show 500 Internal error on the Browser:

``` Java
//spark version in build.gradle compile 'com.sparkjava:spark-core:2.2'
import static spark.Spark.*;

public class App {

    public App() {

         get("/", (req, res) -> {
            int [] r = {1,2,3};
             return r[6];
         });    
    }


    public static void main(String[] args) {        
        new App();
    }

}

```

But with spark 2.5 it will not only prints 500 internal error on the browser and no stack trace on the server/console

``` Java
//spark version in build.gradle compile 'com.sparkjava:spark-core:2.5'
import static spark.Spark.*;

public class App {

    public App() {

         get("/", (req, res) -> {
            int [] r = {1,2,3};
             return r[6];
         });    
    }


    public static void main(String[] args) {        
        new App();
    }

}
```

**Comment by jakaarl** [comment]
Reproduced on fresh 2.5.2-SNAPSHOT.

**Comment by MouettE-SC** [comment]
Duplicate of #486 and fixed in #487
